example/server/classes/Server.ts

```diff
@@ -25,9 +26,11 @@ export interface IPlayer extends MessageEmitter {
 export default class Server extends WebSocketServer {
   constructor(options: IServerOptions, callback?: () => void) {
     super(options, () => {
+
+      MessageBufferPool.enableGlobalScheduler()
```

Comment:
Not a big fan of side-effects like that. Makes it hard to understand when and where things get activated, since it is no longer attached to the code flow.